src/js/components/projects/detail.tsx

```diff
@@ -351,277 +353,298 @@ const ProjectDetail = (
-                  onClick={openCreateEpicModal}
-                  className="slds-m-bottom_x-small tour-create-epic"
-                  disabled={
-                    !tourRunning &&
-                    (!project.has_push_permission || !epics?.fetched)
-                  }
-                />
-                <TourPopover
-                  id="tour-project-create-epic"
-                  align="top left"
-                  body={
-                    <Trans i18nKey="tourCreateEpic">
-                      Create an Epic to make a group of related Tasks. Invite
-                      multiple Collaborators to your Epic and assign people as
-                      Developers and Testers for each Task. Epics are equivalent
-                      to GitHub branches, just like Tasks.
-                    </Trans>
+                    <LabelWithSpinner label={t('Loading Scratch Orgs…')} />
                   }
-                  heading={t('Create Epics to group Tasks')}
+                  disabled
                 />
-              </span>
-              <span className="slds-is-relative metecho-btn-container">
-                <Button
-                  label={t('Create Epic from GitHub Issue')}
-                  variant="outline-brand"
-                  onClick={() => openSelectIssueModal('epic')}
-                  className="slds-m-bottom_x-small tour-create-epic-from-issue"
-                  disabled={
-                    !tourRunning &&
-                    (!project.has_push_permission || !epics?.fetched)
-                  }
-                />
-                <TourPopover
-                  id="tour-project-create-epic-from-issue"
-                  align="top left"
-                  body={
-                    <Trans i18nKey="tourCreateEpicFromIssue">
-                      If you want to help as a Developer on this Project, one
-                      option is to browse the list of GitHub Issues. Issues are
-                      items in GitHub’s bug and enhancement tracking system.
-                      Select an Issue to work on, and create an Epic or Task.
-                      Create an Epic for an Issue if it will require multiple
-                      Tasks to complete. If you’re unsure, begin with a Task and
-                      create an Epic later, as needed.
-                    </Trans>
-                  }
-                  heading={t('Create Epic from GitHub Issue')}
-                />
-              </span>
+              )}
             </div>
-            <EpicTable
-              epics={
-                /* istanbul ignore next */ tourRunning && !epics?.epics?.length
-                  ? [
-                      getDemoEpic({
-                        project: project.id,
-                        github_id: user.github_id,
-                      }),
-                    ]
-                  : epics?.epics || []
+          ) : null
+        }
+      >
+        {project.latest_sha ? (
```

Comment:
@jerivas I'm using the existence of `project.latest_sha` as the indicator that a GH repo has been created. Looking at `models.py` I think that seems reasonable, but do you have concerns about that -- or an alternate suggestion?

Reply:
I think that's a fair assumption 👍